common/app/views/fragments/containers/index.scala.html

```diff
@@ -11,7 +11,7 @@
 @* TODO Consolidate this with the master container template to get rid of repetition *@
 
 @defining(page.webTitle) { case (webTitle) =>
-    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty)"
+    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty, adFree = request.headers.keys.exists(_ equalsIgnoreCase "X-Gu-Commercial-Ad-Free"))"
```

Comment:
By the way @TBonnin I'm not suggesting this is a bad thing to do, but maybe if we do it it should be as a separate PR? I think having a consistent request header construct everywhere would be good in the long run, no?